docker.d/init.sh

```diff
@@ -88,6 +91,13 @@ export WORK_DIR=/app/workdir
 export WORKER_TYPE="${TRUST_DOMAIN}-${TRUST_LEVEL}-${PROJECT_NAME}${WORKER_SUFFIX}"
 export WORKER_GROUP=${WORKER_TYPE}
 export WORKER_ID_PREFIX="${WORKER_TYPE}-"
+export AUTHENTICODE_TIMESTAMP_STYLE=null
+export AUTHENTICODE_CERT_PATH=/app/src/signingscript/data/authenticode_dep.crt
+export AUTHENTICODE_CROSS_CERT_PATH=/app/src/signingscript/data/authenticode_stub.crt
+if [ "$ENV" == "prod" ]; then
+  export AUTHENTICODE_TIMESTAMP_STYLE=old
+  export AUTHENTICODE_CERT_PATH=/app/src/signingscript/data/authenticode_prod.crt
+fi
```

Comment:
is this only different on "prod" environment? "fake-prod" should be the same as "dev"?

Reply:
yes, fake-prod and dev should be the same.
prod signing gets timestamps, and a new cert, as well as different secrets.

Reply:
+1